src/lang_items.rs

```diff
@@ -43,25 +44,8 @@ impl Termination for () {
     }
 }
 
-#[lang = "eh_personality"]
-fn eh_personality() {
-    cycle_leds();
-}
-
-#[lang = "panic_fmt"]
-fn panic_fmt() {
-    flash_all_leds();
-}
-
-fn cycle_leds() {
-    for led in led::all().cycle() {
-        led.on();
-        timer::sleep(Duration::from_ms(100));
-        led.off();
-    }
-}
-
-fn flash_all_leds() {
+#[lang = "panic_impl"]
```

Comment:
It seems this should be `#[panic_implementation]`. Rust development is confusing sometimes.

Reply:
Good point!